src/scripting_engine.c

```diff
@@ -309,3 +310,749 @@ engineMemoryInfo scriptingEngineCallGetMemoryInfo(scriptingEngine *engine,
     engineTeardownModuleCtx(engine);
     return mem_info;
 }
+
+debuggerEnableRet scriptingEngineCallDebuggerEnable(scriptingEngine *engine,
+                                                    subsystemType type,
+                                                    const debuggerCommand **commands,
+                                                    size_t *commands_len) {
+    if (engine->impl.methods.debugger_enable == NULL ||
```

Comment:
Need to check ABI version before accessing these fields. Otherwise it may crash when running a module compiled with valkeymodule.h from Valkey 8.1.

    if (engine->impl.methods.version >= 2) ...